### PR TITLE
Add upstream kernel package

### PR DESCRIPTION
**Pull request by Itxaka**

### CODE REVIEW

File: packages/kernels/upstream/definition.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+category: "kernels"
+name: "linux"
+version: "6.5.7"
+labels:
+  github.repo: "linux"
```
[the remark this answers is not in the document]
Comment (mudler) [reply packages/kernels/upstream/definition.yaml:5]: LTS: https://github.com/mocaccinoOS/desktop/blob/eeeaca01dd36143d82f166d481aa3462791e1ee2/packages/kernels/kernels/mocaccino-lts/kernel/definition.yaml#L21

[the remark this answers is not in the document]
Comment (Itxaka) [reply packages/kernels/upstream/definition.yaml:5]: noice

[the remark this answers is not in the document]
Comment (Itxaka) [reply packages/kernels/upstream/definition.yaml:5]: its even easier now as it has an entry called `latest_stable` :+1:

**Review by mudler — APPROVED** [review]

**Comment by mudler** [comment]
nice!

**Review by mauromorales — APPROVED** [review]

**Pull request merged**